cmd/lakectl/cmd/root.go

```diff
@@ -109,7 +109,11 @@ func getClient() api.ClientWithResponsesInterface {
 	if u.Path == "" || u.Path == "/" {
 		serverEndpoint = strings.TrimRight(serverEndpoint, "/") + api.BaseURL
 	}
+	return serverEndpoint, basicAuthProvider
+}
 
+func getClient() *api.ClientWithResponses {
```

Comment:
We don't need another client func - we can extract the client with response from the client

Reply:
Correct. Fixed.